.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,15 @@
+{
+  "ExpandedNodes": [
+    "",
+    "\\public",
+    "\\src",
+    "\\src\\config",
+    "\\src\\language",
+    "\\src\\lib",
+    "\\src\\lib\\markup",
+    "\\src\\timelines",
+    "\\src\\trials"
+  ],
+  "SelectedNode": "\\public\\favicon.ico",
+  "PreviewInSolutionExplorer": false
+}
```

Comment:
eh seems like since .vs files were already committed before simply adding .vs to gitignore won't change things. anyway, not a big deal it can stay